.gitignore

```diff
@@ -55,6 +55,10 @@ docs/moduleguides
 docs/inmanta.pdf
 docs/_static/*.pdf
 docs/_static/*.png
+docs/lsm/*.png
+docs/lsm/*.pdf
+docs/model_developers/images/*.png
+docs/model_developers/images/*.pdf
```

Comment:
There is a pdf and a png committed at this location. They should either be excluded here or we should first delete them if they are not intended to be tracked.

The same seems to be true for `docs/_static/` but when I delete them git does pick up on it. Any clue how comes those aren't excluded? I don't find any other gitignore with exceptions.

Reply:
My intention behind this change is that when you run `make docs` locally, the local index gets polluted with .png and .pdf files.

Maybe there's another option.

>  they are not intended to be tracked.

I'll look into this on monday

>  Any clue how comes those aren't excluded? 

I have no idea !

Reply:
> The same seems to be true for docs/_static/ but when I delete them git does pick up on it

It looks like git ignores deletions of png/pdf files in docs/_static/ as it should on my machine

Reply:
Very strange. I get
```
	deleted:    docs/_static/architecture.png
	deleted:    docs/_static/inmanta-logo.png
```

Reply:
Hmm, it might be just because we committed them at some point. If I delete them, commit, then put them back, git just ignores them. Perhaps we should add an explicit exception?
```diff
diff --git a/.gitignore b/.gitignore
index e5d41e92..7aeb121a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -55,6 +55,8 @@ docs/moduleguides
 docs/inmanta.pdf
 docs/_static/*.pdf
 docs/_static/*.png
+!docs/_static/architecture.png
+!docs/_static/inmanta-logo.png

 # Mypy
 .mypy
 ```

Reply:
For the ones in `model_developers/` I think we should do the same for the png but we should delete the pdf instead because it is generated from the png. Could you double check that?

Reply:
I could be missing something but as far as I can tell we only use svg to create both pdf and png (https://github.com/inmanta/inmanta-core/blob/aba37a6560a95eda8c703b5bccfebb8b22fbdf98/docs/Makefile#L193-L197)

Reply:
Hmm, you're right. And yet when I delete the pdf and run `make docs` the image is still in the generated pdf.

Reply:
Let's just keep it if we're not sure, but we should add it to the gitignore.